Dockerfile

```diff
@@ -14,15 +14,15 @@
 ##                               BUILD ARGS                                   ##
 ################################################################################
 # This build arg allows the specification of a custom Golang image.
-ARG GOLANG_IMAGE=golang:1.17.5
+ARG GOLANG_IMAGE=golang:1.19.6
```

Comment:
should we compile with the latest golang version like v1.19.11? otherwise there might be cves by Golang

Reply:
I didn't want to use a different image here than what we use for the DISTROLESS_IMAGE which uses `go-runner`

It looks like go-runner doesn't have the latest go versions - https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-build-image/images.yaml#L438-L525

I can update this to 1.19.7 for now, at least, since go-runner has an image for that, but not 1.19.11 unless diverging this image, and the DISTROLESS_IMAGE is fine